### ISSUE TITLE
QA: Premium Subscription payment should be "name your price"

### ISSUE THREAD

**Issue by kaylima** [INN/umbrella-mwcir#15]
1. Go to: http://mwcir-staging.flywheelsites.com/subscription-options/
2. Click on "SUBSCRIBE" under the Premium Subscription option

RESULT: "Pay $100"
![screen shot 2018-07-17 at 3 57 21 pm](https://user-images.githubusercontent.com/15877728/42847801-722a7ff4-89da-11e8-9c59-2707dbd4056c.png)

EXPECT: Should be a "set your own number" for $100 or more. Needs a field to fill out the amount, with a minimum set to $100.

**Comment by kaylima** [comment]
Thanks, @tylermachado ! I wonder if it's because we have 100 filled out and it needs to be blank... but we would definitely want a minimum amount to be set. Curious what their support has to say.

**Comment by kaylima** [comment]
NOTE: we're waiting on support feedback from LP on this. Since it's a nice to have, we'll remove this for now and revisit later if Pam would still like it (and once we have more info from LP.)
